### DISCUSSION TITLE
Ability to merge cities into one megapolis

### DISCUSSION THREAD

**Discussion by morgun** [voxeltycoon/issues#535] [General]
Please add an ability to merge cities into one big entity when their area has some amount of intersection. Like on this screenshot:
![image](https://user-images.githubusercontent.com/1491443/109834218-a144d900-7c4a-11eb-95ff-298eae1844e3.png)

**Comment by greeny** [comment]
What would that be for? How would merged city work differently than a few separate cities?

**Comment by morgun** [comment]
First of all for aesthetic reasons. Seсondly population will also merge and thus business will grow naturally as it's coverage will increase

**Comment by greeny** [comment]
I feel like the combined population would result in less businesses than with separate population.
